### DISCUSSION TITLE
Unable to set WW to standby

### DISCUSSION THREAD

**Discussion by framps** [zivillian/ism7mqtt#139] [Q&A] [answered]
I just tried to set my WW to standby with
`mosquitto_pub -h 192.168.0.12 -t wolf/WWSystem_BM-2_0x35/set/Programmwahl/35012/text -m Standby`

According

```
Example topics to update single properties:

Wolf/<ism7 ip address>/WWSystem_BM-2_0x35/set/Programmwahl/35012/value
Wolf/<ism7 ip address>/WWSystem_BM-2_0x35/set/Programmwahl/35012/text
```
this should work. But unfortunately it doesn't work for me :-(

I used
```
./findExistingISM7MQTTparameters.sh programmwahl
>>> "Programmwahlschalter gemeinsam": BM-2 - 220108
>>> "BUSCONFIG_Programmwahlschalter-Gemeinsam": BM-2 - 220130
>>> "Programmwahl": DHK_BM-2 - 340029
>>> "Programmwahl": DHK_BM-2 - 340032
>>> "Programmwahl": WWSystem_BM-2 - 350012
>>> "Programmwahl": WWSystem_BM-2 - 350015
```
to detect the correct ids but

neither 350012 nor 350015 works.

Would be great to get my Wolf system to accept any modifications. What do I miss?

**Comment by zivillian** [comment]
Hmm... Bad documentation...

The path you tried is only used for duplicate parameter names. Normally the path does not include the parameter id. You can check at which topic the current value is reported and just insert the `set/` after the device.
I guess it would be `wolf/WWSystem_BM-2_0x35/set/Programmwahl/text`

**Reply by framps** [reply]
> Bad documentation...

NP. Doc is the last thing a developer cares of :wink: 

Unfortunately  `mosquitto_pub -h 192.168.0.12 -t wolf/WWSystem_BM-2_0x35/set/Programmwahl/text -m Standby` doesn't work. I also tried `standby` as argument - just to make sure there is no case issue - but it doesn't work :-(

**Reply by framps** [reply, marked as answer]
Thank you very much for your hint. Actually there was a major typo in the topic I used: I used `wolf` instead of `Wolf` :cry: 

I finally managed to toggle WW status from off to auto with mosquitto:

```
mosquitto_pub -h 192.168.0.12 -t Wolf/192.168.0.156/WWSystem_BM-2_0x35/set/Programmwahl/350012/value -m 1
```

and

```
mosquitto_pub -h 192.168.0.12 -t Wolf/192.168.0.156/WWSystem_BM-2_0x35/set/Programmwahl/350012/value -m 0
```
